docs/api_reference/auth_api/introduction.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 1
+---
+
+# Introduction
+
+Auth API allows to authenticate from external APIs.
```

Comment:
This is not 100% correct, right? Since authorization is implicit in the token. Maybe this should be `verify that an API token is valid`?

Reply:
```suggestion
Auth API allows to verify that the provided API token is valid and ready to be used.
```